src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -272,6 +251,7 @@ const useCoreFeed = () => {
     onDeleteBlockedPostCompleted,
     saveSearchHeight,
     sendViewPostTime,
+    sendViewPostTimeWithFeeds,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `sendViewPostTimeWithFeeds` is being returned from the hook. This allows it to be used in other parts of the application, promoting code reusability.
```diff
+ sendViewPostTimeWithFeeds,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->